proto/rill/runtime/v1/resources.proto

```diff
@@ -341,6 +342,10 @@ message SecurityRuleRowFilter {
   Expression expression = 3; // Regular query expression referencing dimension names
 }
 
+message SecurityRuleTransitiveAccess {
+  ResourceName resource = 1; // this rules resolves to rules that provides access to whatever is needed to access this resource. Currently only used for reports and alerts.
```

Comment:
Would be nice also to support explores initially. (Explores are quite simply and would ensure backwards compatibility with old public URLs, but canvases we can do in a separate PR as they don't currently have public URLs.)

Reply:
sure